unittests/snapshot_tests.cpp

```diff
@@ -446,6 +445,17 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_compatible_versions, SNAPSHOT_SUITE, snapshot
    }
    // This isn't quite fully automated.  The snapshots still need to be gzipped and moved to
    // the correct place in the source tree.
+   // -------------------------------------------------------------------------------------------------------------
+   // Process for supporting a new snapshot version in this test:
+   // ----------------------------------------------------------
+   // 1. update `current_version` and the list of versions in `for` loop
+   // 2. run: `unittests/unit_test -t "snapshot_tests/test_com*" -- --save-snapshot` to generate new snapshot files
+   // 3. copy the newly generated files (see `ls -lrth ./unittests/snapshots/snap_*` to `leap/unittests/snapshots`
+   //    for example `cp ./unittests/snapshots/snap_v7.* ../unittests/snapshots`
+   // 4. edit `unittests/snapshots/CMakeLists.txt` and add the `configure_file` commands for the 3 new files.
+   //    now the test should pass.
+   // 5. add the 3 new snapshot files in git (beware the top `.gitignore` hides the `snapshots` directory - why???)
```

Comment:
Ask this guy: https://github.com/AntelopeIO/leap/pull/1460

Reply:
> Ask this guy: #1460

I asked him. His memory is not what it used to be, he probably should rely on `git blame` to avoid asking silly questions. Still he suggested a remedy which I just checked in.

Reply:
lol